package/Dockerfile.agent

```diff
@@ -19,6 +19,8 @@ RUN curl -sLf https://github.com/rancher/loglevel/releases/download/${LOGLEVEL_V
 ARG VERSION=dev
 LABEL io.cattle.agent true
 ENV AGENT_IMAGE rancher/rancher-agent:${VERSION}
+# For now, this value needs to be manually synced with the one in the main Dockerfile. This pins downstream webhook's version.
+ENV CATTLE_RANCHER_WEBHOOK_VERSION=2.0.5+up0.3.5-rc7
```

Comment:
Do we also want to 0 out the `CATTLE_RANCHER_WEBHOOK_MIN_VERSION` like we do in the main dockerfile? Might help out users who have done the manual workaround.

Reply:
In the main Dockerfile, we have `ENV CATTLE_RANCHER_WEBHOOK_MIN_VERSION=''`, which, I believe, is the same as not having a value at all, so I don't think there would be a difference if we don't include this line in agent's Dockerfile.

Reply:
Will that be an issue for upgraded setups? For example, say that I am currently running 2.7.3 and upgrade to this version. Since we have made the MIN_VERSION override the exact version, this will use the min_version logic, and not upgrade the webhook. Do I have that right?

Reply:
Discussed offline - we need to also zero out the env var in agent's Dockefile, as in the main Dockerfile.